src/metabase/models/permissions.clj

```diff
@@ -1079,6 +1079,49 @@
     (when (and has-native-perms? (not (group-has-native-perms? group-or-id database-or-id)))
       (grant-native-readwrite-permissions! group-or-id database-or-id))))
 
+; Audit Permissions helper fns
+
+(defenterprise default-audit-db-id
+  "OSS implementation of `audit-db/default-audit-db-id`, which is an enterprise feature, so does nothing in the OSS
+  version."
+  metabase-enterprise.audit-db [] ::noop)
+
+(defenterprise default-audit-collection-entity-id
+  "OSS implementation of `audit-db/default-audit-collection-entity-id`, which is an enterprise feature, so does nothing in the OSS
+  version."
+  metabase-enterprise.audit-db [] [])
+
+(defn update-audit-collection-permissions
+  "Will remove or grant audit db (AppDB) permissions, if the instance analytics permissions changes."
+  [group-id changes]
+  (when config/ee-available?
```

Comment:
Can this be a `defenterprise` function instead, with the implementation in an EE namespace, to ensure it has the correct license? You could maybe make a new `metabase-enterprise.audit-app.permissions` namespace